src/plugins/vis_type_xy/public/utils/domain.ts

```diff
@@ -21,7 +21,7 @@ export const getXDomain = (params: Aspect['params']): DomainRange => {
   const minInterval = (params as DateHistogramParams | HistogramParams)?.interval ?? undefined;
 
   if ((params as DateHistogramParams).date) {
-    const bounds = getTimefilter().getActiveBounds();
+    const bounds = getTimefilter().getBounds();
```

Comment:
@nickofthyme that was changed to use `getActiveBounds` due to the following:
https://github.com/elastic/kibana/pull/89822#issuecomment-775864605

Reply:
Oh, so any idea why this would return `undefined` when a time range is active?

Reply:
Ok I manually enabled the time filter for date histogram charts. I'm not sure why this is not already happening automatically. But I see timelion is doing this and I modeled the renderer changes after timelion. See 52b7290